### PR TITLE
Use mainClass instead of removed main

### LINKED ISSUE #46: Can't run with Gradle 8
Running with 0.8.14 I get the following exception:

```
* Exception is:
java.lang.NoSuchMethodError: 'org.gradle.api.tasks.JavaExec com.github.erizo.gradle.JcstressTask.setMain(java.lang.String)'
        at com.github.erizo.gradle.JcstressPlugin.addJcstressTask(JcstressPlugin.java:225)
        at com.github.erizo.gradle.JcstressPlugin.apply(JcstressPlugin.java:114)
        at com.github.erizo.gradle.JcstressPlugin.apply(JcstressPlugin.java:64)
        at org.gradle.api.internal.plugins.ImperativeOnlyPluginTarget.applyImperative(ImperativeOnlyPluginTarget.java:43)
        at org.gradle.api.internal.plugins.RuleBasedPluginTarget.applyImperative(RuleBasedPluginTarget.java:51)
        at org.gradle.api.internal.plugins.DefaultPluginManager.addPlugin(DefaultPluginManager.java:187)
        at org.gradle.api.internal.plugins.DefaultPluginManager.access$100(DefaultPluginManager.java:52)
        at org.gradle.api.internal.plugins.DefaultPluginManager$AddPluginBuildOperation.run(DefaultPluginManager.java:282)

```

### PR DESCRIPTION
**Pull request by carl-mastrangelo**
Fixes #46

### CODE REVIEW

File: src/main/java/com/github/erizo/gradle/JcstressPlugin.java [hunk 1]
```diff
@@ -222,7 +222,7 @@ private JcstressTask addJcstressTask() {
         final JcstressTask jcstressTask = project.getTasks().create(TASK_JCSTRESS_NAME, JcstressTask.class);
 
         jcstressTask.dependsOn(jcstressJarTask);
-        jcstressTask.setMain("org.openjdk.jcstress.Main");
```
Comment (jerzykrlk) [inline src/main/java/com/github/erizo/gradle/JcstressPlugin.java:225]: Good evening Carl.

Thanks for the Pull Request! Let me check this within a few days. I'll need to verify which lowest Gradle version has this method. 

Thank you again!

Comment (carl-mastrangelo) [reply src/main/java/com/github/erizo/gradle/JcstressPlugin.java:225]: FWIW I tried to see when the mainClass attribute was added, but it's not well documented. Sorry for the chore!

Comment (jerzykrlk) [reply src/main/java/com/github/erizo/gradle/JcstressPlugin.java:225]: Hello Carl.

Now I see that the tests were disabled.

I fixed the problem i na bit different way - for now it'll work. Could you review these changes?

https://github.com/reyerizo/jcstress-gradle-plugin/pull/48

Thanks!

**Comment by jerzykrlk** [comment]
Fixed with https://github.com/reyerizo/jcstress-gradle-plugin/pull/48

**Pull request closed without merging**